tests/unit/steps/test_download_accessions.py

```diff
@@ -24,9 +25,9 @@ def setUp(self):
                 ]
             ],
             output_files=["assembly/nr.refseq.fasta"],
-            output_dir_local="/mnt/idseq/results",
+            output_dir_local=tempfile.TemporaryDirectory().name,
             output_dir_s3="s3://dummy_bucket",
-            ref_dir_local="/mnt/idseq/ref/",
+            ref_dir_local=tempfile.TemporaryDirectory().name,
```

Comment:
I know this wasn't here originally but would you mind adding tempfile cleanup? For example:

````python
with tempfile.TemporaryDirectory() as output_dirname:
    # ...
```

Reply:
This happens automatically on process exit.